src/handlers/web3.ts

```diff
@@ -698,53 +703,51 @@ export const getDataForNftTransfer = async (
   from: string,
   to: string,
   asset: ParsedAddressAsset
-): Promise<string> => {
-  const nftVersion = asset.asset_contract?.nft_version;
-  const schema_name = asset.asset_contract?.schema_name;
-  if (nftVersion === '3.0') {
```

Comment:
this condition obsolete, `nftVersion` is an nft property provided by opensea, which we no longer use. not sure entirely what it did, it is not documented– i'm guessing the condition equates to "if the nft abides by the current erc721 standard". erc721 has changed over time, so some nfts that are classified as an erc721 don't actually adhere to the currently defined standard